fix: restructure the API section (#449)
safe-global/safe-docs: pages/core-api/overview.mdx

import { Cards, Card } from 'nextra/components'
import Guides from '../../assets/svg/guides.svg'
import Reference from '../../assets/svg/reference.svg'

# Safe\{Core\} API

The Safe\{Core\} API offers REST endpoints to keep track of transactions sent via Safe contracts. It also offers endpoints to send transactions to allow offchain collecting of signatures or informing the owners about a pending transaction to be sent to the blockchain.

> We also offer a TypeScript client for the API as part of the [API Kit](../sdk/api-kit) in the Safe\{Core\} SDK.

Reviewer: ```suggestion
We also offer the [API Kit](../sdk/api-kit), a TypeScript client for the Safe Transaction Service API in the Safe\{Core\} SDK.
```
